src/sdk-reference/js/6/getting-started/node-js/index.md

```diff
@@ -0,0 +1,210 @@
+
+<div class="alert alert-info">
+There are also more <a href="{{ site_base_path }}guide/1/essentials/installing-kuzzle/">alternative ways</a> to install Kuzzle.
+</div>
+
+This command downloads, installs and runs Kuzzle.
+
+Use the `--no-run` option to prevent the script from running Kuzzle.
+
+Once the installation process is complete, you will see the following message:
+
+```bash
+# Kuzzle is now running
+```
+
+Your Kuzzle is now running! To test it, you can explore the main HTTP API by clicking this <a href="http://localhost:7512?pretty">link</a> or by using cURL on the command line:
+
+```bash
+curl "http://localhost:7512/?pretty"
+```
+
+If everything is working you should see a JSON document that contains a list of API endpoints.
+
+<div class="alert alert-success">
+Congratulations! You have completed the Kuzzle installation, it will now accept requests on <code>localhost:7512</code>:
+<ul>
+  <li>via <strong>HTTP</strong></li>
+  <li>via <strong>Websocket</strong></li>
+</ul>
+</div>
+
+<div class="alert alert-info">
+Having trouble?
+<ul>
+  <li>Get in touch with us on <a href="https://gitter.im/kuzzleio/kuzzle">Gitter!</a> We're happy to help.</li>
+  <li>Try one of <a href="{{ site_base_path }}guide/1/essentials/installing-kuzzle/">these</a> alternative installation methods.</li>
+</ul>
+</div>
+
+## Fun with the SDK
+
+It's time to play with the [Kuzzle Javscript SDK]({{ site_base_path }}sdk-reference/js/6). In this section, we will store a document and subscribe to notifications in Kuzzle using the Javascript SDK.
+
+Before proceeding, please make sure your system has **Node.js** version 8 or higher (<a href="https://nodejs.org/en/download/">instructions here</a>) installed.
+
+## Prepare your environment
+
+Create your playground directory and install the [Javascript SDK]({{ site_base_path }}sdk-reference/js/6) from the command line using npm:
+
+```sh
+mkdir "kuzzle-playground"
+cd "kuzzle-playground"
+npm install git://github.com/kuzzleio/sdk-javascript.git#6-beta
+```
+
+<div class="alert alert-info">
+If you are performing a clean install you might see some `UNMET PEER DEPENDENCY` warnings, these are safe to ignore as they refer to optional dependencies.
+</div>
+
+Then, create an `init.js` file and start by adding the code below.  
+This will load the Kuzzle Javascript SDK and then instantiate a client that will connect to Kuzzle via websockets.  
+
+[snippet=load-sdk]
+
+<div class="alert alert-info">
+You need to replace 'kuzzle' which is the Kuzzle backend hostname with 'localhost' or the hostname where your Kuzzle backend is running.
+</div>
+
+Next we add a listener to be notified in case of connection error:
+
+```javascript
+kuzzle.on('networkError', error => {
+  console.error('Network Error: ', error);
+});
+```
+
+Then we have to connect the client to your Kuzzle server with the `connect()` method.  
```

Comment:
```suggestion
Then, connect the client to your Kuzzle server with the `connect()` method.  
```

Reply:
This sentence should make clearer that it actually explains what the next code snippet does, and it should be merged with the next sentence to prevent users thinking that a code example is missing (it was my 1st assumption)